camel-core/src/main/java/org/apache/camel/model/ThrottleDefinition.java

```diff
@@ -43,6 +44,8 @@
 public class ThrottleDefinition extends ExpressionNode implements ExecutorServiceAwareDefinition<ThrottleDefinition> {
     // TODO: Camel 3.0 Should not support outputs
 
+    @XmlElement(name = "correlationExpression")
+    private ExpressionSubElementDefinition correlationExpression;
```

Comment:
This correlationExpression must be put after to element, you had to find a way to setup the right order for it.  I don't think you can do it by just extends the ExpressionNode.

Reply:
Not sure if it is a must. We can impose order with xmlorder tag. I did try. It did not help.

Reply:
Yeah I would also put correlationExpression as 2nd, eg its a new option, and it makes more sense, as its optional.

Reply:
this is addressed